app/src/main/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityEffectHandler.kt

```diff
@@ -8,24 +8,23 @@ import org.simple.clinic.facility.FacilityRepository
 import org.simple.clinic.patient.PatientRepository
 import org.simple.clinic.patient.PatientSearchResult
 import org.simple.clinic.util.scheduler.SchedulersProvider
+import javax.inject.Inject
```

Comment:
We did not need this commit in the current PR. We should've created a different PR for maintaining this. Merging this now.

Reply:
Actually, we had for this 😅 . With Mobius update, the older approach was not working for this, so I had to change it to our latest implementations.